### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by A7medAmine** [opensourcepos/opensourcepos#3967] [Q&A]
HELLO GUYS 
i just wanna know how to edit the shortcurts  in the sell section can you help me plz .
and also THANKS a lot for your work its incredible how this app works 
![Screenshot 2024-04-13 054026](https://github.com/opensourcepos/opensourcepos/assets/140296603/fd28d8e9-94c3-41e0-bd9b-df1e38371128)

**Comment by jekkos** [comment]
@WebShells you know this part best

**Reply by A7medAmine** [reply]
I already did it i suggest you use the f1 f2 ... Keys on the shortcuts alt+7 and the other ones are just too much work
